app/views/posts/index.html.erb

```diff
@@ -1,23 +1,28 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>All Posts by User</title>
+  <title>All Posts</title>
 </head>
 <body>
-  <h1>All Posts by User</h1>
+  <h1>All Posts</h1>
   
-  <ul>
-    <% @posts.each do |post| %>
-    <li>
-      <% if post.author %>
-        <%= link_to post.title, user_post_path(user_name: post.author.name, id: post.id) %>
-      <% else %>
-        <!-- Handle the case where author is nil (e.g., display "Unknown author") -->
-        Unknown author: <%= post.title %>
-      <% end %>
-    </li>
+  <% @users.each do |user| %>
+    <div>
+      <h2><%= user.name %></h2>
+      <p>Number of Posts: <%= user.posts.count %></p>
+      
+      <h3>List of Posts:</h3>
+      <ul>
+        <% user.posts.each do |post| %>
+          <li>
+            <strong>Title:</strong> <%= link_to post.title, user_post_path(user.name, post.id) %><br>
+            <strong>Number of Comments:</strong> <%= post.comments.count %><br>
+            <strong>Likes:</strong> <%= post.likes.count %>
+          </li>
+        <% end %>
+      </ul>
+    </div>
```

Comment:
- [ ] Good job so far 🙌 However, please make sure to use `recent_comments` method present in `post model` to render 5 most recent comments in the `app/views/posts/index.html.erb` page as we can see from the wireframe provided that only 5 most recent comments are rendered on the post index page. Also, please make sure to follow the given wireframe for style. You can customize your app the way you want but please make sure to follow the general layout. 
- [ ] Displaying all users on post index page is not in accordance with the wireframe provided. Please fix this as well ✨⛏
- [ ] Currently, I am getting the same data for both users. Please have a look at the following images for this bug 🐛 Please make sure to fix this

This is what I have for first user | This is what I have for second user
---| ---| 
![image](https://github.com/Kidd254/BlogApp/assets/106119105/7fe0ea6a-7cba-44a4-9a4e-87555a5a8842) | ![image](https://github.com/Kidd254/BlogApp/assets/106119105/f3997281-8e13-4abd-9203-28d6945f4b7d)



Current | Wireframe
---| ---|
![image](https://github.com/Kidd254/BlogApp/assets/106119105/8b659d85-ce03-4a4d-96a8-5c29dd7cf51d) | ![image](https://github.com/Kidd254/BlogApp/assets/106119105/0b727af9-e002-4d5b-bd64-4d43f7a54174)